greenpithumb/greenpithumb.py

```diff
@@ -72,6 +74,15 @@ def pump_if_needed(self, _):
     ]
 
 
+def make_camera_poller(photo_interval, image_path, record_queue):
+    local_clock = clock.LocalClock()
+    poller_factory = poller.SensorPollerFactory(local_clock, photo_interval,
+                                                record_queue)
```

Comment:
(Optional) Feels strange to create a new poller factory just for the camera poller. My first thought was to just add a ```poll_interval``` argument to ```create_camera_poller```, but that also doesn't feel good.